### DISCUSSION TITLE
SNAP to MintPy: how to subset  to make the input data have the same row and column?

### DISCUSSION THREAD

**Discussion by gaiyf** [insarlab/MintPy#981] [Q&A]
Hello, i wonder to know how to subset the input data in SNAP to make the input data have the same row and column?

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).
